test/e2e/kubelet_perf.go

```diff
@@ -180,10 +172,60 @@ var _ = Describe("Kubelet [Serial] [Slow]", func() {
 			name := fmt.Sprintf(
 				"for %d pods per node over %v", podsPerNode, monitoringTime)
 			It(name, func() {
-				runResourceTrackingTest(framework, podsPerNode, nodeNames, rm, itArg.limits)
+				numNodes := nodeNames.Len()
+				totalPods := podsPerNode * numNodes
+				rcConfig := &RCConfig{
+					Client:    framework.Client,
+					Name:      fmt.Sprintf("resource%d-%s", totalPods, string(util.NewUUID())),
+					Namespace: framework.Namespace.Name,
+					Image:     "gcr.io/google_containers/pause:2.0",
+					Replicas:  totalPods,
+				}
+				runResourceTrackingTest(framework, rcConfig, nodeNames, rm, itArg.limits)
+			})
+		}
+	})
+	Describe("stressed resource usage tracking", func() {
+		rTests := []resourceTest{
+			{podsPerNode: 35,
+				probePeriodInSeconds: 1,
+				limits: containersCPUSummary{
+					"/kubelet":       {0.50: 0.60, 0.95: 0.35},
+					"/docker-daemon": {0.50: 0.60, 0.95: 0.30},
+				},
+			},
+		}
+		for _, testArg := range rTests {
+			itArg := testArg
+			podsPerNode := itArg.podsPerNode
+			name := fmt.Sprintf(
+				"for %d pods per node over %v", podsPerNode, monitoringTime)
+			It(name, func() {
+				numNodes := nodeNames.Len()
+				totalPods := podsPerNode * numNodes
+				probe := &api.Probe{
+					Handler: api.Handler{
+						HTTPGet: &api.HTTPGetAction{
+							Port: intstr.FromInt(80),
+							Path: "/",
+						},
+					},
+					PeriodSeconds: itArg.probePeriodInSeconds,
+				}
+
+				rcConfig := &RCConfig{
+					Client:         framework.Client,
+					Name:           fmt.Sprintf("resource%d-%s", totalPods, string(util.NewUUID())),
+					Namespace:      framework.Namespace.Name,
+					Image:          "gcr.io/google_containers/test-webserver",
+					Replicas:       totalPods,
+					ReadinessProbe: probe,
```

Comment:
I think it depends on what the goal is. Adding a liveness probe will effectively double the number of probes (and the probe to container ratio). This would probably exaggerate the probe overhead, which might be a good thing? (Assuming it scales linearly).